### PR TITLE
Support iOS 13 features (such as Dark Mode)

### PR DESCRIPTION
**Pull request by uuttff8**
## Changes in this pull request
Support Swift 5
Support Dark Mode
Add SVProgressHud as expected
Add named colors

Issue fixed: 
-

## Issues in this pull request
Markdown in proposal still light mode only

### Checklist

- [x] Project builds and runs as expected.
- [x] No new linting violations have been introduced.
- [ ] No new bugs have been introduced.
- [ ] I have reviewed the [contributing guide](https://github.com/evolution-app/ios/blob/development/.github/CONTRIBUTING.md)
- [ ] I'm attaching to this PR some screenshots (if applicable)

### CODE REVIEW

**Comment by unnamedd** [comment]
Hi @uuttff8, thanks a lot for your PR!
As soon as possible I will review in detail your changes, quickly looking I didn’t see any problem. 

Thanks to bring the dark mode to the project! 

I really appreciated your effort!

**Comment by uuttff8** [comment]
Screen Shot 2019-11-28 at 9 58 20 PM
now navigation title color is ok

**Comment by unnamedd** [comment]
@uuttff8,  do not forget to add yourself as a contributor [here](https://github.com/Evolution-App/iOS/blob/development/README.md) (README) and [here](https://github.com/Evolution-App/iOS/blob/development/swift-evolution/sourcecode/controllers/about/main/AboutViewController.swift#L64) (About section).

**Comment by uuttff8** [comment]
@unnamedd all done!

File: swift-evolution/sourcecode/controllers/about/main/AboutViewController.swift [hunk 1]
```diff
@@ -63,6 +63,7 @@ final class AboutViewController: UITableViewController {
     
     private func contributors() -> Section {
         let members = [
+            Contributor(text: "Anton Kuzmin", type: .github, value: "babnikbezbab"),
```
Comment (unnamedd) [inline swift-evolution/sourcecode/controllers/about/main/AboutViewController.swift]: @uuttff8 I guess you wrote your Twitter username here... 😁

Comment (uuttff8) [reply swift-evolution/sourcecode/controllers/about/main/AboutViewController.swift]: thank you!

**Pull request merged**